intro-to-code/readme.md

```diff
@@ -0,0 +1,119 @@
+# Introduction to Code
+
+In today's digital age, computers have become an integral part of our lives. They power our smartphones, tablets, laptops, and countless other devices. They manage our finances, help us stay connected with friends and family, and even assist us in learning new skills like coding. Before we dive into the world of coding, it's essential to grasp the basics of how computers work and the profound influence they have on our daily lives.
+
+## Learning objectives
+
+By the end of this lesson, you should be able to:
+
+- Describe the basics of how computers work.
+- Identify some of the ways in which code impacts our everyday life.
+- Describe what a "coder" does and why it is so useful.
+- Describe the process of learning to code and how this program will feel different than school.
+
+---
+
+## Fundamentals of computing
+
+A computer, in its essence, is a complex machine designed to process and manipulate information rapidly. It achieves this by executing a sequence of instructions given to it, and these instructions are conveyed in a language known as a "programming language." To understand how these languages work and how they communicate with computers, we must delve deeper into the intricacies of computer programming.
+
+### How computers process information
+
+Computers have two primary components that collaborate to process information:
+
+1. **Central Processing Unit (CPU)**: The CPU serves as the computational powerhouse of the computer. It is responsible for executing instructions, performing calculations, and managing data. To communicate with the CPU effectively, programmers write instructions in a programming language, which is then translated into a form that the CPU can understand and execute. This intermediary step is critical, as CPUs only comprehend a specific set of binary instructions, composed of 0s and 1s.
+
+1. **Memory**: Memory plays a crucial role in the computer's operation by temporarily storing data and instructions required by the CPU. Two types of memory are particularly significant:
+
+   - **RAM (Random Access Memory)**: This volatile memory provides fast but temporary storage. When you run a program, its instructions and data are loaded into RAM for quick access by the CPU. However, once the computer is turned off or restarted, the data in RAM is erased.
+
+   - **Storage Drives**: These include hard drives and solid-state drives (SSDs) and offer long-term storage for data and programs. Unlike RAM, the data stored on storage drives remains intact even when the computer is powered down.
+
+<details><summary>❗️ How does writing code influence a computer's actions?</summary>
+ As described above, code is the way in which people can influence a computer's actions. Writing code allows people to tell a computer what to do in a way that it will understand. A person could write code that executes a series of actions or stories something in memory.
+</details>
+
```

Comment:
I think two extra things to consider:

Adding info and differentiating human interfaces - keyboards, mice, screens, from the CPU/memory. Especially in the days of smartphones, I think it is worth separating the brick into a couple more pieces. 

Computers work by switches that translate into ones and zeros. I think this info is always in the ether though never quite explained anywhere, and will give them a touchpoint when you talk about compilers and computer languages - the things humans understand vs computers. 

Otherwise, no other notes! Great lesson!